common/src/traversal.rs

```diff
@@ -185,8 +186,7 @@ impl<'a> RayTraverser<'a> {
                     // because the AABB check can have false positives.
                     let ray_node_distance = (next_node_transform * self.ray.position).w.acosh();
                     let ray_length = tanh_distance.atanh();
-                    if ray_node_distance - ray_length - self.radius > dodeca::BOUNDING_SPHERE_RADIUS
-                    {
+                    if ray_node_distance - ray_length - self.radius > BOUNDING_SPHERE_RADIUS {
```

Comment:
Please keep this in the original form, `dodeca::BOUNDING_SPHERE_RADIUS`.

Reply:
But:

```rs
error[E0433]: failed to resolve: use of undeclared crate or module `dodeca`
   --> common/src/traversal.rs:188:71
    |
188 | ...radius > dodeca::BOUNDING_SPHERE_RADIUS {
    |             ^^^^^^ use of undeclared crate or module `dodeca`
    |
help: consider importing this module
    |
1   + use crate::dodeca;
```

And it doesn't see the imports for some reason:
```rs
use crate::{
    collision_math::Ray,
    dodeca::{Side, Vertex, BOUNDING_SPHERE_RADIUS},
    graph::{Graph, NodeId},
    math,
    node::ChunkId,
    proto::Position,
};
```

Reply:
(I tried this on the command line, not VSCode.)

Reply:
I'll look into this in a couple of days.

Reply:
You'll need to undo your change to the import as as well. In `traversal.rs`, the original import from `dodeca` was `dodeca::{self, Side, Vertex},`. The `self` is what allowed `dodeca` to resolve.